nodes/Arguments.py

```diff
@@ -1,2 +1,25 @@
+"""
+Arguments astroid node
+
+Attributes:
+    - args         (list)  list of non-keyword argument names
```

Comment:
As we discussed last week, use `List` instead of `list`, and in fact if you can be more specific with the node types, please do so. E.g., `List[Name]` (if that's the right node type).